### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by kpiyush04**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [ ] CHANGE-LOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by orbeckst** [comment]
What issue are you referencing? You checked the box but there's no issue number (e.g., #1234)  If you're referring to something else, add a link in the comment of your issue report (edit the issue report).

Please add some notes to your PR comment to give some context: describe your changes and your reason for making them.

**Comment by kpiyush04** [comment]
Sure I'll do it.
Thanks for the review.

File: README.rst [hunk 1]
```diff
@@ -2,13 +2,13 @@
   MDAnalysis Repository README
 ================================
 
-|numfocus| |build| |cov| [*]_
+|numfocus| |build| |cov| |mdanalysis| [*]_
```
Comment (orbeckst) [inline README.rst]: We don't need the extra badge.

File: README.rst [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
+MDAnalysis_ is an object-oriented Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists.
```
Comment (orbeckst) [inline README.rst]: We started de-emphasizing the "object oriented", so it's ok not to have it here.

File: README.rst [hunk 3]
```diff
@@ -21,6 +21,13 @@ a `tax-deductible donation`_ to help the project pay for developer time, profess
   :target: https://numfocus.org/project/mdanalysis
   :align: center
   :alt: NumFOCUS
+
```
Comment (orbeckst) [inline README.rst]: We don't need our logo here.

If anything, a smaller NumFOCUS logo would be nice. I am not 100% sure if the size parameter works with the rst rendered on GitHub, but that's something worthwhile trying.

File: README.rst [hunk 4]
```diff
@@ -50,6 +57,34 @@ Example analysis script
 There are a number of tutorials_ on the MDAnalysis homepage that explain
 how to conduct RMSD calculations, Alignment and many more features of MDAnalysis.
 
+Example Trajectories-analysis script
```
Comment (orbeckst) [inline README.rst]: We intentionally kept the README small and we don't want to add more examples. Don't include it.

**Comment by orbeckst** [comment]
As stated in my review, there's not much in this PR that we want to use at the moment, so I am closing it.

Thank you for your suggestions.

Please have a look at our other issues and our documentation (look at https://www.mdanalysis.org/mdanalysis/ for the current docs). There might well be a number of easy fixes in there.

**Pull request closed without merging**